api/Console.json

```diff
@@ -231,19 +237,19 @@
               "version_added": "8.3.0"
             },
             "opera": {
-              "version_added": true
+              "version_added": "55"
             },
             "safari": {
-              "version_added": null
+              "version_added": "13"
```

Comment:
This change is also part of https://github.com/mdn/browser-compat-data/pull/6526 which increases my confidence that it's right.